### PR TITLE
Add Beam and Beam Testnet

### PR DESCRIPTION
**Pull request by federava**
All contracts were deployed via de singleton factory:

- [https://github.com/safe-global/safe-deployments/pull/296](https://github.com/safe-global/safe-deployments/pull/296)
- [https://github.com/safe-global/safe-singleton-factory/issues/214](https://github.com/safe-global/safe-singleton-factory/issues/214)
- [https://github.com/safe-global/safe-singleton-factory/issues/215](https://github.com/safe-global/safe-singleton-factory/issues/215)

Thanks in advance!

### CODE REVIEW

**Comment by federava** [comment]
Hi @ElvisKrop @nick8319, adding Beam mainnet and testnet as they were added to [safe-deployments here](https://github.com/safe-global/safe-deployments/pull/296) and I did not see here [https://github.com/safe-global/safe-eth-py/pull/773](https://github.com/safe-global/safe-eth-py/pull/773).

Please let us know of any further requirements. Thank you!

**Comment by falvaradorodriguez** [comment]
Hi @federava , please comment "I have read the CLA Document and I hereby sign the CLA" to accept the CLA. Thanks

**Comment by federava** [comment]
I have read the CLA Document and I hereby sign the CLA

File: gnosis/eth/ethereum_network.py [hunk 1]
```diff
@@ -803,6 +803,8 @@ class EthereumNetwork(Enum):
     RIKEZA_NETWORK_TESTNET = 12715
     QUANTUM_CHAIN_TESTNET = 12890
     SPS = 13000
+    CREDIT_SMARTCHAIN_MAINNET = 13308
```
Comment (falvaradorodriguez) [inline gnosis/eth/ethereum_network.py:806]: This would not be necessary. It is repeated in the next two lines.

**Review by falvaradorodriguez — CHANGES_REQUESTED** [review]

File: gnosis/eth/ethereum_network.py (hunk 1 above)
Comment (federava) [reply gnosis/eth/ethereum_network.py]: Removed

**Review by falvaradorodriguez — APPROVED** [review]

**Review by falvaradorodriguez — CHANGES_REQUESTED** [review]

**Comment by federava** [comment]
Hi @falvaradorodriguez, updated branch and did the linting.

Thank you

**Review by falvaradorodriguez — APPROVED** [review]

**Pull request merged**